### ISSUE TITLE
does not support gds.gz format

### ISSUE THREAD

**Issue by Larson-Li** [heitzmann/gdstk#222]
In actual project, the GDS file is large, so compress the gds to gds.gz is common actions, but the gdstk.read_gds does not support the action.

**Comment by Larson-Li** [comment]
Does gdstk will support gz compress format when using gdstk.read_gds API?

**Comment by heitzmann** [comment]
Not yet. It's a planned feature but there's no one working on it at this time.

**Comment by Larson-Li** [comment]
@heitzmann Is there schedule of this feature?

**Comment by heitzmann** [comment]
Not really. This feature is not trivial and I don't really know when I'll have the time to dedicate to it. I'm always open to PR's, though! Just make sure it is implemented in a C-style (not modern C++) to keep in line with the rest of the code base.

**Comment by Larson-Li** [comment]
> Look at my pull request #286 where I have added this feature But I need help testing it

Gds and gz are golden formats, sample test is enough for merging. I will test and feedback after is approved
